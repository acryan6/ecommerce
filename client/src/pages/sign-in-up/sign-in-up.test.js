import React from "react";
import { shallow } from "enzyme";
import SignInAndUp from "./sign-in-up.component";

describe("SignInAndUp page", () => {
  it("should render component", () => {
    expect(shallow(<SignInAndUp />)).toMatchSnapshot();
  });
});
