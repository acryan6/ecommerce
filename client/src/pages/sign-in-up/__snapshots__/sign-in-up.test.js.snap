// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInAndUp page should render component 1`] = `ShallowWrapper {}`;
